ONSdigital/response-operations-ui
# Tbl133 use CI selection classifier correctly

- Get form_type from file name and upload CI using this form type classifier
- Use classifier to calculate expected CI

To test functionality:
Use classifier to calculate expected CI
1) `Make up` ras-rm-docker-dev
2) For RSI/Bricks/blocks surveys load `business-survey-sample-data.csv` sample file from `rasrm-acceptance-tests/resources/sample_files` using this response-operations-ui branch
3) The preview panel should show 5 businesses and 1 Collection Instrument (this is because CI classifier for these surveys is FORM_TYPE)
4) Do the same for BRES and you should see 5 businesses and 5 Collection Instrument in the sample preview panel.
**You may need to run this sql script because the data for BRES may not be correct:
            `update ras_ci.seft_instrument`
`set file_name = 'test_collection_instrument.xlsx'`
`where file_name is null`

Get form_type from file name and upload CI using this form type classifier
1) `Make up` ras-rm-docker-dev
2) For RSI/Bricks/blocks surveys load `064_201803_0001.xlsx` collection instrument from `rasrm-acceptance-tests/resources/collection_instrument_files` using this response-operations-ui branch
3) Check response operations makes a post request to backstage with param 'classifiers' with correct 'form_type' value from the file (0001 in this case). 


A text change in this PR will mean a test in https://github.com/ONSdigital/rasrm-acceptance-tests will break. https://github.com/ONSdigital/rasrm-acceptance-tests/pull/77 is the fix for that and will need to be merged in after this PR is in
